voluncheer/profiles/forms/organizations.py

```diff
@@ -26,8 +27,16 @@ def save(self, commit=True):
         )
         return user
 
+
+""" OrganizationChangeForm
```

Comment:
This should go just below the class definition rather than above it. The first line should also give a brief description of the class, the name is obvious from the definition.

Here and elsewhere.